release-notes.txt

```diff
@@ -1,3 +1,632 @@
+RIOT-2021.01 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+The 2021.01 release includes:
+ - Device Firmware Upgrade support for USBUS
+ - USB HID minimal implementation
+ - IEEE 802.15.4 security initial implementation
+ - 6LowPAN Minimal Forwarding and Selective Fragment Recovery initial
+   implementation
+ - Initial support for QN808x, SAMD10 STM32MP1 and STM32L5 CPUs
+ - Initial support for Cortex-M33
+ - A module for selecting entropy sources
+ - Initial support for the Decawave Impulse Radio-Ultra Wideband (UWB Core)
+ - Capability to join or leave link layer multicast groups (GNRC)
+ - Support for IN865 region in GNRC LoRaWAN
+ - 83 additional modules supported by Kconfig
+ - 14 new boards, 6 new device drivers, 8 packages updated
+
+In order to improve community interaction, during this release cycle there was
+some work to migrate from a mailing list based communication to forum-based
+communication. We are proud to present the RIOT-OS forum available at
```

Comment:
```suggestion
In order to improve community interaction, during this release cycle some work
was done work to migrate from a mailing-list-based communication to forum-based
communication. We are proud to present the RIOT-OS forum available at
```

Reply:
hmmm reading twice, I think it would maybe sound better like this:
```suggestion
In order to improve community interaction, during this release cycle there
were some efforts to migrate from a mailing list based communication to
forum-based communication. We are proud to present the RIOT-OS forum
available at
```

Reply:
"there were some efforts" sounds like we tried a bit, but did not really progress any more than having the test platform running (so somewhat of an understatement)...

```suggestion
In order to improve community interaction, during this release cycle we
started to migrate from a mailing-list-based communication to
forum-based communication. We are proud to present the RIOT-OS forum
available at
```

Reply:
(please fix line endings independent from this suggestion, some lines might be to short with this suggestion...)

Reply:
done (beded39ffbf465648d2867273684b09fe2e39b2a)